examples/bri-3/src/bri/workgroup/workflows/agents/workflowsStorage.agent.ts

```diff
@@ -104,4 +106,24 @@ export class WorkflowStorageAgent {
       where: { id: workflow.id },
     });
   }
+
+  async storeWorkflowTransaction(
+    bpiAccountCandidate: BpiAccount,
+    workflowCandidate: Workflow,
+  ): Promise<any> {
+    const boundStoreNewWorkflow = this.storeNewWorkflow.bind(this);
+    const boundStoreNewBpiAccount =
+      this.bpiAccountStorageAgent.storeNewBpiAccount.bind(
+        this.bpiAccountStorageAgent,
+      );
+    const results = await this.prisma.executeTransaction(() => {
```

Comment:
you are still passing only one operation here, so not quite showing array of sequential operations, it should be array [storeBpiAccount, storeWorkflow], not storeBpiAccount.then(storeWorkflow)? this way it seems we are controlling this execution, and prisma should sequentially execute all or nothing behind the scenes if i understand correctly